lab_Arrays/src/main/java/comper/Main.java

```diff
@@ -0,0 +1,32 @@
+package comper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Main {
+    private static final Logger log = LogManager.getLogger(Math.class);
+    public static void main(String[] args) {
+        ArrayList<Europa> list = new ArrayList<Europa>();

```

Comment:
Note !! prefer to use the most suitable abstraction when declaring types for variables methods  - for example, List in this case, to able to replace realization ArrayList<->LinkedList<->AnotherList